src/index.ts

```diff
@@ -119,17 +119,67 @@ function makePassesFilter(opts: FilterOpts): (msg: Msg) => boolean {
 class threads {
   private readonly ssb: Record<string, any>;
   private readonly isBlocking: (obj: any, cb: CB<boolean>) => void;
+  private readonly isFollowing: ((obj: any, cb: CB<boolean>) => void) | null;
 
   constructor(ssb: Record<string, any>, _config: any) {
     this.ssb = ssb;
     this.isBlocking = ssb.friends?.isBlocking
       ? ssb.friends.isBlocking
       : IS_BLOCKING_NEVER;
+    this.isFollowing = ssb.friends?.isFollowing;
     this.ssb.db.registerIndex(HashtagsPlugin);
   }
 
   //#region PRIVATE
 
+  private maybeOnlyKeepFollowingMessages =
+    (enabled: boolean) => (source: any) =>
+      pull(
+        source,
+        pull.asyncMap((msg: Msg, cb: CB<Msg | null>) => {
+          if (!enabled) {
+            cb(null, msg);
+            return;
+          }
+
+          if (!this.isFollowing) {
```

Comment:
Maybe we can remove this check, and instead put it early inside `public()` and `publicSummary()` as an error. I'll put a comment down there suggesting how.

Reply:
good suggestion - only thing is that TS will complain when this is removed so it'd be needed in one form or another within this function